django_measurement/models.py

```diff
@@ -80,7 +81,7 @@ def get_default_unit(self):
             return unit_choices[0][0]
         return self.measurement.STANDARD_UNIT
 
-    def from_db_value(self, value, expression, connection, context):
```

Comment:
Since we're not using the extra args/kwargs; maybe it would be easier to just change the signature to `def from_db_value(self, value, *args, **kwargs)`?

Reply:
Right, more elegant solution